packages/amplify-category-storage/provider-utils/awscloudformation/service-walkthroughs/s3-walkthrough.js

```diff
@@ -384,5 +384,50 @@ function convertToCRUD(parameters, answers) {
   }
 }
 
+function getIAMPolicies(resourceName, crudOptions) {
+  let policy = {};
+  let actions = new Set();
 
-module.exports = { addWalkthrough, updateWalkthrough, migrate };
+  crudOptions.forEach((crudOption) => {
```

Comment:
In askReadWrite function, we have 'create/update' instead of 'create' and 'update'.